conda_build/render.py

```diff
@@ -31,6 +31,7 @@
 from conda.gateways.disk.create import TemporaryDirectory
 from conda.models.records import PackageRecord
 from conda.models.version import VersionOrder
+from conda_package_handling.api import extract
```

Comment:
This API [can only extract `.conda` or `.tar.bz2`,](https://github.com/conda/conda-package-handling/blob/main/src/conda_package_handling/api.py) not any other `.tgz` etc. variant.